### DISCUSSION TITLE
Auto discord joiner

### DISCUSSION THREAD

**Discussion by Eimis54** [Vedza/NitroSniperGo#52] [Q&A] [answered]
Hey man, I found your auto-joiner and wanted to ask one thing, Is it possible to make it join giveaways in certain servers, or to not join in certain servers or chats

**Comment by Vedza** [comment, marked as answer]
Hey,
Unfortunately this is not possible.

**Reply by Youniz** [reply]
It is possible to do it. But this is gonne get ur accounts disabled pretty fast. So i don't recommend adding this feature

**Reply by Vedza** [reply]
Actually I think I misread this message.
If you want the sniper to not join giveaway on some servers you can add them in blacklists in settings.json and this will do what you want
